src/django_icalendar/migrations/0001_calendar_model.py

```
@@ -0,0 +1,62 @@
+# Generated by Django 6.0.2 on 2026-03-02 02:01
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+    initial = True
+
+    dependencies = []
```

Ruff (0.15.2)

[warning] Mutable default value for class attribute

(RUF012)